spec.html

```diff
@@ -4393,6 +4393,21 @@ <h1>The Lexical Environment and Environment Record Specification Types</h1>
       <p>The Lexical Environment and Environment Record types are used to explain the behaviour of name resolution in nested functions and blocks. These types and the operations upon them are defined in <emu-xref href="#sec-lexical-environments"></emu-xref>.</p>
     </emu-clause>
 
+    <emu-clause id="sec-abstract-closure">
+      <h1>The Abstract Closure Specification Type</h1>
+      <p>The <dfn>abstract closure</dfn> specification type is used to refer to algorithm steps together with a collection of values. Abstract closures are meta-values and are invoked using function application style such as _closure_(_arg1_, _arg2_). Like abstract operations, invocations perform the algorithm steps described by the abstract closure.</p>
+      <p>In algorithm steps that create an abstract closure, values are captured with the verb "capture" followed by a list of aliases. The value associated with each alias at creation time is captured. In steps that specify the algorithm to be performed when an abstract closure is called, each captured value is referred to by the alias that was used to capture the value.</p>
```

Comment:
Re "The value associated with each alias at creation time": There's the possibility of misreading this as talking about the creation of the *alias*. So I think it's better to start with "When the abstract closure is created".

Re "is captured": I don't think the passive voice helps here. Is there a problem with saying that the abstract closure does the capturing?

Re "steps that specify the algorithm to be performed when an abstract closure is called": You said you'd adopt "in the abstract closure's algorithm steps".

```suggestion
      <p>In algorithm steps that create an abstract closure, values are captured with the verb "capture" followed by a list of aliases. When the abstract closure is created, it captures the value that is associated with each such alias at that time. In the abstract closure's algorithm steps, each captured value is referred to by the alias that was used to capture the value.</p>
```

Reply:
Sure, will adopt.